package.json

```diff
@@ -55,7 +55,7 @@
     "dev-graphiql": "yarn workspace graphiql dev",
     "e2e": "yarn run e2e:build && yarn workspace graphiql e2e",
     "e2e:build": "yarn build && yarn workspace graphiql build-bundles",
-    "eslint": "NODE_OPTIONS=--max-old-space-size=4096 eslint --max-warnings=0 --ignore-path .gitignore --cache .",
+    "eslint": "NODE_OPTIONS=--max-old-space-size=4096 eslint --max-warnings=0 --ignore-path .gitignore --ignore-path .prettierignore --cache .",
```

Comment:
Eslint doesn't accept multiple ignore-path as prettier v3 do

Reply:
ok! then it makes sense to switch to rename the file to `.eslintignore`, and to add it to the prettier `--ignore-path`

Reply:
@B2o5T do you approve of this strategy? anything that can go wrong? trying to keep it simple